src/components/routeComponents/memberRegistrationCorrectionForm/CorrectionsForm.jsx

```diff
@@ -173,6 +179,41 @@ const CorrectionsForm = ({
 							{renderSubmitButtons()}
 						</Col>
 					</Row>
+					{isWarningMessage && (
```

Comment:
Can we rename this to
```suggestion
					{showChangesLostWarning && (
```